.jscsrc

```diff
@@ -50,6 +50,8 @@
 		"examples/typescript-*/js/**/*.js",
 		"examples/vanilladart/**/*.js",
 		"examples/vanilla-es6/dist/bundle.js",
+		"examples/react-redux/js/reducers/*.js",
```

Comment:
These reducers uses object spread operator.

Adding the `esnext` option to jscs works but I couldn't find a way to extend a jscs configuration or use a specific one on the `js` folder in order to override this option. I also tried disabling jscs with inline comments (`/* jscs: disable */`) and didn't work. So I removed reducers from linting :|